tests/src/python/test_python_repr.py

```diff
@@ -147,6 +147,12 @@ def testQgsMimeDataUri(self):
         d.providerKey = 'my_provider'
         self.assertEqual(d.__repr__(), "<QgsMimeDataUtils::Uri (my_provider): my_uri>")
 
+    def testQgsMapLayerRepr(self):
+        vl = QgsVectorLayer(
+            'Point?crs=epsg:4326&field=pk:integer&field=cnt:integer&field=name:string(0)&field=name2:string(0)&field=num_char:string&key=pk',
+            'my funky name', 'memory')
```

Comment:
Can you make the text ã̷̡̪͓̟̰͕̻̹͖̃̏̎̾̐̚͜ ̵̢͕̋̒̂̍͗̎̿͐̕b̷̢͓͓̥̫̓̎i̵̦͓͑̇t̶̳̟̖̣͓̩̅̏͂̏ ̶̘͎͚̻̊͂̑̈́́̚͝ṁ̵̨͓̱͉̫͔̰̊̅́̀͐̅͆͐̚͜o̴͙̠̗̅͑r̵͓̫̘̫̭̓̆͘ë̴̛͔̩̟͉͈́ ̷̰̲̣̹͕̰̮̄̾͜f̷̢̝̱͈̹̹͕̋͊͐͑̌͒̈́͊̈́͜u̶̧͒ͅn̶̘̯̤̤͖̹̔ḱ̸̨ÿ̶̮̖̖̱̻́͗͒̊  to check unicode support?